### DISCUSSION TITLE
Feedback and suggestions wanted! Help us improve Mermaid-js and Mermaid Chart with your feedback

### DISCUSSION THREAD

**Discussion by DominicDeWalt** [mermaid-js/mermaid#5142] [General]
Hi everyone! 

I’m Dominic and I recently joined the authors of Mermaid at Mermaid Chart as a product manager! I'm super stoked to dive into this community and connect with all of you. As I get started, I am hoping to learn more about how and why all of you are using Mermaid so that I can help improve the product by directly building features that you actually want! 

 If you’re willing to share your feedback, please book a [30 minute conversation](https://calendly.com/dominic-01w/feedback) with me so that you can directly influence Mermaid’s and Mermaid Chart’s future development.

If you don’t want to chat live but still want to help us out, please give your thoughts in this google form here: https://forms.gle/XqVs57cdgmPXLusq7

Thank you all in advance! If you have any questions then I’d love to chat with you in the comments here too. Have a great day!

**Comment by huynhicode** [comment]
Thanks, @DominicDeWalt !  🙌
